### DISCUSSION TITLE
Help to custom file associations

### DISCUSSION THREAD

**Discussion by tutods** [mallowigi/a-file-icon-idea#193] [General]
Hi
Anyone can help me to create a custom file associations?
Already try, but not working

**Comment by mallowigi** [comment]
What's your file association?

**Reply by tutods** [reply]
I can't find the svg files on Atom icons plugin to set. I want set icon to ormconfig file (for example).

**Reply by mallowigi** [reply]
Download the icons here: https://github.com/mallowigi/iconGenerator

**Reply by tutods** [reply]
And where I put? I can delete after set?

**Reply by mallowigi** [reply]
Just download this in your disk, then in the custom file association settings, select the desired icon
